test/test_masked.py

```diff
@@ -313,6 +313,7 @@ def test_mask_layout(self, layout, device, dtype, op, sample_inputs):
             expected = op.op(r_inp, *r_args, **r_kwargs)
             self.assertEqualMasked(actual, expected, outmask)
 
+    @skipIfTorchDynamo("Test does not work with TorchDynamo")
```

Comment:
If you file a tracking issue it's nice to link to it for context for peeps in the future

Reply:
yes